kit/src/routes/__layout.svelte

```diff
@@ -0,0 +1,92 @@
+<script context="module" lang="ts">
+	import { base } from "$app/paths";
+
+	export async function load(input: LoadInput) {
+		if (prerendering || !import.meta.env.DEV) {
```

Comment:
The issue with the previewing previously was the lack of `|| !import.meta.env.DEV`.

Because `prerendering` would be false on the client side, even if the html pages themselves were prerendered, as there was still some client side javascript

Reply:
great catch!